templates/takeovers/_fing_webinar.html

```diff
@@ -0,0 +1,46 @@
+<section data-lang="en" lang="en-gb" class="p-strip is-deep p-takeover--fing-webinar js-takeover {% if  default %}u-hide{% endif %}" {% if default %}data-default="true"{% endif %}>
```

Comment:
If the background is dark you should add `is-dark` class which will adjust so the text to white.

Reply:
it was .p-strip--dark